packages/dd-trace/src/appsec/telemetry/index.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+const logs = require('./api/logs-plugin')
+const metrics = require('./api/metrics-plugin')
+const { Verbosity, isDebugAllowed, isInfoAllowed, parseVerbosity } = require('./verbosity')
+const {
+  add,
+  drain,
+  initTelemetryCollector,
+  getTelemetryCollectorFromContext,
+  GLOBAL
+} = require('./telemetry-collector')
+
+const TRACE_METRIC_PATTERN = '_dd.instrumentation_telemetry_data.appsec'
+
+class Telemetry {
+  configure (config) {
+    this.enabled = config.telemetry.enabled
+
+    if (this.enabled) {
+      this.verbosity = parseVerbosity(config.iastTelemetryVerbosity)
+      if (this.verbosity !== Verbosity.OFF) {
+        metrics.init(config, drain)
+      }
+      logs.init(config)
+    }
+  }
+
+  stop () {
+    this.enabled = false
+
+    drain()
+    metrics.stop()
+    logs.stop()
+  }
+
+  isEnabled () {
+    return this.enabled
+  }
+
+  isDebugEnabled () {
+    return this.isEnabled() && isDebugAllowed(this.verbosity)
+  }
+
+  isInformationEnabled () {
+    return this.isEnabled() && isInfoAllowed(this.verbosity)
+  }
+
+  increase (metric, tag, context) {
+    add(metric, 1, tag, context)
+  }
+
+  add (metric, value, tag, context) {
```

Comment:
It's a bit confusing having two different methods with same name and same arguments...

Reply:
true, changed